tests/playwright-test/web-server.spec.ts

```diff
@@ -602,27 +601,3 @@ test('should treat 3XX as available server', async ({ runInlineTest }, { workerI
   expect(result.output).toContain('[WebServer] listening');
   expect(result.output).toContain('[WebServer] error from server');
 });
-
-test('should be able to kill process that ignores SIGTERM', async ({ runInlineTest }, { workerIndex }) => {
-  test.skip(process.platform === 'win32', 'there is no SIGTERM on Windows');
-  const port = workerIndex + 10500;
-  const result = await runInlineTest({
-    'test.spec.ts': `
-      const { test } = pwt;
-      test('pass', async ({}) => {});
-    `,
-    'playwright.config.ts': `
-      module.exports = {
-        webServer: {
-          command: 'node ${JSON.stringify(SIMPLE_SERVER_THAT_IGNORES_SIGTERM_PATH)} ${port}',
```

Comment:
Should we replace it with a test that checks graceful termination in response to sigterm or is it covered elsewhere already?

Reply:
All other tests check it!